versions.sh

```diff
@@ -87,9 +87,13 @@ for version in "${versions[@]}"; do
 	for suite in \
 		bullseye \
 		buster \
+		alpine3.17 \
 		alpine3.16 \
-		alpine3.15 \
 	; do
+		# https://github.com/docker-library/php/pull/1348
+		if [ "$suite" = 'alpine3.17' ] && [[ "$version" = 8.0* ]]; then
```

Comment:
Just a minor swap of the conditional here (so it's testing for "Alpine that _isn't_ 3.16" instead of explicitly "Alpine 3.17" so that 3.18+ doesn't require explicit care to DTRT; also we keep `$rcVersion` with `-rc` stripped for exactly these types of comparisons :eyes:): :+1:

```suggestion
		if [ "$rcVersion" = '8.0' ] && [[ "$suite" = alpine* ]] && [ "$suite" != 'alpine3.16' ]; then
```

Reply:
That looks a lot better, thanks for the suggestion! I do think Alpine 3.18+ will still require some manual intervention, as at that point you'll need 3+ active versions in `versions.sh` (the two most recent versions + keep 3.16 for 8.0), so you also want to add an ignore 3.16 clause for >= 8.1 at that point.

Reply:
:smirk: no OpenSSL 3 support?  no more Alpine! :innocent:

(mostly joking - yeah, we probably have to revisit this later anyhow, but it makes me feel a lot better to have the conditional reflect that it's 3.17+ that will likely default to OpenSSL 3 rather than having to remember to revisit this bit explicitly :smile:)